packages/core-components/src/components/product-list/ProductCollapse.tsx

```diff
@@ -0,0 +1,93 @@
+import { BlueCheck, WhiteCheck } from '@/icons';
+import React, {useState} from 'react';
+
+type ProductItemProps = {
+  productName: string;
+  productImage: string;
+  productDescription: string;
+};
+
+export type ProductListProps = {
+  products: Array<ProductItemProps>;
+};
+
+const ProductCollapse: React.FC<ProductItemProps> = ({ 
+  productName, 
+  productImage, 
+  productDescription,
+}) => {
+  const [closeCollapse, setCloseCollapse] = useState(true);
+  const [navColor, setNavColor] = useState('bg-[#FFE1E1]');
+  const [nameColor, setNameColor] = useState('text-[#002651]');
```

Comment:
why do we need these 2 as states here, em. Can we just do `closeCollapse ? bg-[#002651] : bg-[#FFE1E1]`

Reply:
and `toggleCollapse` will just flip the closeCollapse bool